tests/reliability/test_cluster/test_cluster_logs/test_cluster_worker_logs_order/data/Integrity_sync.yml

```diff
@@ -20,15 +20,9 @@
   tag: 'Updating local files: Start.*'
 - log_id: log13
   parent: log12
-  tag: 'Received [0-9]* missing files to update from master.'
+  tag: 'Updating local files: End.'
```

Comment:
The order was change according the new behavior introduced in https://github.com/wazuh/wazuh/pull/19888#pullrequestreview-1706253248.